### PR TITLE
Fix #7074: impactt app

### LINKED ISSUE #7074: sirepo app for Impact-T
Needs the lattice element views and configuration in the schema.
And the lattice items in the impactt.js config()

### PR DESCRIPTION
**Pull request by gurhar1133**

### CODE REVIEW

File: sirepo/package_data/static/json/impactt-schema.json [hunk 1]
```diff
@@ -92,73 +144,52 @@
...
             "radius": ["Radius [m]", "RPNValue", 0],
             "x_offset": ["Misalignment X [m]", "RPNValue", 0],
             "y_offset": ["Misalignment Y [m]", "RPNValue", 0],
             "x_rotation": ["Rotation Error X [rad]", "RPNValue", 0],
             "y_rotation": ["Rotation Error Y [rad]", "RPNValue", 0],
             "z_rotation": ["Rotation Error Z [rad]", "RPNValue", 0]
         },
-        "DRIFTTUBE": {
-            "zedge": ["Z Edge", "RPNValue", 0, "Longitudinal position of the entrance face of the element"]
+        "BPM": {
```
Comment (moellep) [inline sirepo/package_data/static/json/impactt-schema.json:165]: Remove the BPM element and replace with a set of elements which have a negative ele_type from here. Just the common items for now, it doesn't have to include all of them.
https://github.com/ChristopherMayes/lume-impact/blob/master/impact/parsers.py#L400

For example, different models for OFFSET_BEAM (name, l, x_offset, y_offset, py_offset, z_offset, pzoffset), WRITE_BEAM (name, l, filename, sample_frequency)

File: sirepo/package_data/static/json/impactt-schema.json [hunk 2]
```diff
@@ -186,31 +217,225 @@
...
+            "title": "DRIFTTUBE",
+            "description": "The DRIFTTUBE element defines a drift space.",
+            "fieldsPerTab": 8,
+            "advanced": [
+                "name"
+            ]
+        },
+        "DIPOLE": {
+            "title": "DIPOLE",
+            "description": "A Dipole",
```
Comment (moellep) [inline sirepo/package_data/static/json/impactt-schema.json:357]: No need to have a "description" field here if it is just restating the name of the element displayed in the panel heading.

**Review by moellep — CHANGES_REQUESTED** [review]

**Comment by gurhar1133** [comment]
@moellep ping

**Review by moellep — APPROVED** [review]
I made some changes with a76d2c6 - please give it a test and merge if it looks OK.
- Use RPNValue instead of Float in the schema
- Added default-data.json and a few changes to template.impactt so new sims can be created
- Updated the example to use the element positions from the lattice

**Pull request merged**